ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -376,10 +351,5 @@ initNetwork registry nodeArgs kernel RunNetworkArgs{..} = do
       (Just (Socket.SockAddrInet6 0 0 (0, 0, 0, 1) 0))
```

Comment:
Not related to this PR, but the IPv6 address looks wrong. ::1 is the loop back address, IPv6's version of 127.0.0.1.

Reply:
@karknu aren't they the same:
```
Prelude Network.Socket> SockAddrInet6 0 0  (0,0,0,1) 0
[::1]:0
```

Reply:
Ach right, I see what you mean, it should be `(0,0,0,0)` here.